im.dino.Dino.yml

```diff
@@ -0,0 +1,50 @@
+---
+app-id: im.dino.Dino
+runtime: org.gnome.Platform
+runtime-version: "42"
+sdk: org.gnome.Sdk
+command: dino
+finish-args:
+  - "--share=ipc"
+  - "--socket=x11" # for whatever reason breaks with fallback-x11 on GNOME on Wayland
```

Comment:
maybe because it doesn't support wayland?

Reply:
This is still unanswered

Reply:
as i'm not the developer, i don't know the technical reason. But i want to keep the comment so that if in future wayland is supported the manifest can be updated easily

Reply:
I don't see any upstream issue about missing wayland support. As it is a GTK 3 application, it should work just fine under wayland.

Reply:
Works just fine here with `--nosocket=x11 --socket=fallback-x11` on GNOME Wayland.

Reply:
Thanks I'm following your instructions